docs/BasicCommunication/SetAppProperties/assets/SetAppProperties.txt

```diff
@@ -0,0 +1,47 @@
+title SetAppProperties 
+
+participant Local Web App
+participant SDL
+participant HMI
+
+==<align:center>Pre-conditions: 1. SDL and HMI are started==
+
+opt SetAppProperties for a new WebEngine app 
+
+HMI -> SDL: BC.SetAppProperties(policyAppID:"123", enabled:true)
+SDL-->HMI: BC.SetAppProperties(SUCCESS)
+SDL->HMI: BC.OnAppPropertiesChange()
+SDL->HMI: BC.UpdateAppList(deviceID)
+HMI-->SDL: BC.UpdateAppList(SUCCESS)
+HMI->SDL: SDL.ActivateApp()
+SDL-->HMI: SDL.ActivateApp response(SUCCESS)
+HMI->SDL: BC.OnAppActivated
+
+note over HMI: WebEngine app is displayed\nin the main window on the HMI
+
+SDL->Local Web App: OnHMIStatus(hmiLevel, windowID)
+end
+
+opt SetAppProperties - Changes to properties of existing WebEngine app
+HMI -> SDL: BC.SetAppProperties(policyAppID:"123", enabled:true, **nicknames: {"Hello Sdl1"}**)
+SDL-->HMI: BC.SetAppProperties(SUCCESS)
+SDL->HMI: BC.OnAppPropertiesChange()
+end
+
+opt SetAppProperties (enabled=false)
+HMI -> SDL: BC.SetAppProperties(policyAppID:"123", enabled:false)
+SDL-->HMI: BC.SetAppProperties(SUCCESS)
+SDL->HMI: BC.OnAppPropertiesChange()
+end
+
+opt SetAppProperties with unknown parameter
+HMI -> SDL: BC.SetAppProperties(unknown parameter, policyAppID:"123",enabled:true)
+SDL ->SDL: Cut off the unknown parameter
+
+note over SDL:Process the request as assigned
+end
+
+opt SetAppProperties with missed mandatory parameters/with invalid parameters
+HMI -> SDL: BC.SetAppProperties(nicknames:HelloSDL)
```

Comment:
```suggestion
HMI -> SDL: BC.SetAppProperties(nicknames:"HelloSDL")
```
already is missing mandatory param, if you want to show both cases (missing param and invalid param) lets use a number for nicknames

Reply:
please, see updates in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/187/commits/4ebba070bd78b95c3a00721cc2bcb639a6dfb422